python/beeai_framework/errors.py

```diff
@@ -51,12 +50,17 @@ def __get_message(error: Exception) -> str:
     def is_retryable(error: Exception) -> bool:
         """is error retryable?."""
         if isinstance(error, FrameworkError):
-            return error._is_retryable
+            return error.is_retryable
+        # TODO: Is a CancelledError (asyncio) retryable? Any other error classes
         return isinstance(error, CancelledError)
```

Comment:
i believe CancelledError/AbortError should not retried... see:
- https://github.com/i-am-bee/beeai-framework/blob/main/typescript/src/errors.ts#L108-L116
- https://github.com/i-am-bee/beeai-framework/blob/59156f0d5c9b57dbf5288bd7263c597d5bbd510f/python/beeai_framework/errors.py#L113

in which case this should be
```suggestion
        return not isinstance(error, CancelledError)
```

Reply:
That was my thought. updating.